torchbenchmark/models/detectron2_maskrcnn/__init__.py

```diff
@@ -0,0 +1,84 @@
+import torch
+import os
+import itertools
+import random
+import numpy as np
+
+# TorchBench imports
+from torchbenchmark.util.model import BenchmarkModel
+from torchbenchmark.tasks import COMPUTER_VISION
+
+# setup environment variable
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
+DATA_DIR = os.path.join(CURRENT_DIR, ".data", "coco2017-minimal")
+if not 'DETECTRON2_DATASETS' in os.environ:
+    os.environ['DETECTRON2_DATASETS'] = DATA_DIR
+
+from detectron2.config import instantiate
+from detectron2 import model_zoo
+from detectron2.utils.collect_env import collect_env_info
+from detectron2.utils.logger import setup_logger
+from detectron2.utils.events import EventStorage
+
+torch.manual_seed(1337)
+random.seed(1337)
+np.random.seed(1337)
+torch.backends.cudnn.deterministic = True
```

Comment:
why deterministic? wondering will that hurt performance.

Also, will these global settings affect or get affected by other benchmarks in this repo?

Reply:
Sorry this is a bug, both `cudnn.deterministic` and `cudnn.benchmark` should be set to False.
We run each benchmark in separate processes so they won't interfere with each other.